src/components/Card.js

```diff
@@ -1,13 +1,16 @@
 import React, { useState } from 'react';
+import { useEffect } from 'react/cjs/react.development';
 
 function Card(props) {
   const [saveCardPopup, setSaveCardPopup] = useState(false);
   const [mobileMode, setMobileMode] = useState();
   const [marked, setMarked] = useState(false);
 
-  window.addEventListener('resize', () => {
-    setMobileMode(window.innerWidth);
-  });
+  useEffect(() => {
+    window.addEventListener('resize', () => {
```

Comment:
`window.addEventListener('resize', () => {`

please, delete `window.addEventListener('resize', () => {` from all components and leave it only in `App`. You need to make `const [screenWidth, setScreenWidth] = useState(window.innerWidth);` only in `App` and pass `screenWidth` to the components that need that info. You can't add such listseners to `window` when you create each card. It will create 100 listeners if you add 100 cards